collector/Cargo.toml

```diff
@@ -44,6 +44,7 @@ regex = "1.7.1"
 analyzeme = "12.0.0"
 
 benchlib = { path = "benchlib" }
```

Comment:
Could you please add this above this? I tried to visually separate local and remote dependencies (database and intern are moved as well in https://github.com/rust-lang/rustc-perf/pull/1961).